generator/src/components/FormCheckbox/index.svelte

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+  export let name: string
+  export let label: string = ""
+</script>
+
+<div class="wrapper">
+  <div class="checkbox">
+    <input type="checkbox" id={name} />
+    <label for={name} style="--size: 20px">
+      <svg viewBox="-10,-10,70,70">
+        <path d="M5 30 L 20 45 L 45 5" />
+      </svg>
```

Comment:
I would generally prefer these to be in file format because it's easier to see what it is from the filename, and easier to update

Reply:
<img width="568" alt="image" src="https://github.com/ioxiocom/ioxio-tags/assets/137529618/0570f24d-0a29-4f6e-8412-610085e3b24f">
We are updating the check svg via CSS.

Reply:
yes I saw that, you can still do the same by putting it in a component, it might need `:global()` selector then